JWBestPracticeApps/Captions/Captions/ViewController.swift

```diff
@@ -38,17 +38,31 @@ class ViewController: UIViewController, PlayerCaptionsDelegate {
         
         /// Open a `do-catch` block to handle possible errors with the builders.
         do {
-            /// First, use the `JWPlayerItemBuilder` to create a `JWPlayerItem` that will be used by the player configuration.
+            var captions = [JWMediaTrack]()
+            
+            /// First, create a captions track. This example demonstrates using a local URL rather than a remote one.
```

Comment:
Something strange here. Why /// and not // ?
/// is for markdown for header docs. Regular comments use //

Reply:
@s-seibert I am following the pattern established in the rest of the file, where triple-slashes are used. I am happy to purge them!

Reply:
Done, btw